app/models/users/suspended.rb

```diff
@@ -0,0 +1,11 @@
+module Users
+  class Suspended < ApplicationRecord
+    self.table_name = "suspended_users"
```

Comment:
This table name makes more sense on the DB layer but doesn't reflect the model's namespacing so I explicitly set it.

Reply:
I'm almost conflicted with `users_suspended` which wold keep it close to `users` in the list of tables:

```text
| public   | user_blocks                          | table    
| public   | user_subscriptions                   | table    
| public   | users                                | table    
| public   | users_gdpr_delete_requests           | table    
| public   | users_roles                          | table    
```

Reply:
The problem with `users_suspended` that it breaks Rails convention by not being pluralized. Not that it matters much since I'm already manually specifying the username, but I like consistency.

So in the end I decided to go for the most explicit name possible, the model is now called `Users::SuspendedUsername` and the corresponding table is `users_suspended_usernames`.

It's a bit verbose, but this isn't used in many places, so I think it's ok.